d2bs/kolbot/libs/common/Pather.js

```diff
@@ -900,6 +901,15 @@ ModeLoop:
 				if (me.area === 40 && npc && getDistance(me, npc) < 50) {
 					if (npc && npc.openMenu()) {
 						Misc.useMenu(0x0D37);
+
+						while (me.area !== 1) {
```

Comment:
can you change this to 
 if (!Misc.poll(function() {
    if (me.area === 1) {
         return true;
    }

    return false;
}, 2000, 100)) {
    throw new Error("Failed to go to act 1 using Warriv");
}